src/client/src/ui/spotTile/components/SpotTile.tsx

```diff
@@ -20,6 +20,23 @@ const TileHeaderWrapper = styled.div`
 `
 
 export default class SpotTile extends PureComponent<Props> {
+  /**
+   * In order to integrate Glue42 with channels the clicked symbol needs to be published to the channel.
+   * @param {string} symbol
+   */
+  publishToChannel = (symbol: string) => {
```

Comment:
Glue42 refs should not be made in the main application.


Additionally, Interactions with glue42 belong in a middleware, not components.

Reply:
What would be a correct way to create a middleware for those interactions, is there an existing convention?